conjurapi/host_factory.go

```diff
@@ -5,9 +5,12 @@ import (
 	"fmt"
 	"github.com/cyberark/conjur-api-go/conjurapi/response"
 	"net/url"
+	"strings"
 	"time"
 )
 
+const HostFactory = ":host_factory:"
```

Comment:
This constant makes the code below a bit confusing so I think it would be better to just include the string in-line since I think it's only used in one place?